cartographer_ros/cartographer_ros/offline_node.cc

```diff
@@ -269,9 +272,7 @@ void RunOfflineNode(const MapBuilderFactory& map_builder_factory) {
           ? playable_bag_multiplexer.PeekMessageTime()
           : ros::Time();
   while (playable_bag_multiplexer.IsMessageAvailable()) {
-    if (!::ros::ok()) {
-      return;
-    }
+    CHECK(::ros::ok());
```

Comment:
Changed these to checks, `!ros::ok()` should be considered as an error here IMO.

Reply:
I put that there. This path is taken if you ctrl-C.